app_update/lib/src/controller/update_controller.dart

```diff
@@ -253,7 +180,9 @@ class UpdateController extends UpdateControllerBase {
     await _availableUpdateStream.close();
   }
 
-  Future<UpdateConfig> _findUpdateConfig({
+  // TODO название так себе, но лучше я не придумал
+  Future<List<AppUpdate>> _findAppUpdatesFromConfig({

```

Comment:
Мы вроде везде термин Update используем, мб App лишнее

```suggestion
  Future<List<AppUpdate>> _findUpdatesFromConfig({

```